src/backend/electron/engineAndVvppController.ts

```diff
@@ -217,7 +224,8 @@ export class EngineAndVvppController {
       log.info(
         "All ENGINE process kill operations done. Running post engine kill process",
       );
-      return this.vvppManager.handleMarkedEngineDirs();
+      const vvppManager = getVvppManager();
+      return vvppManager.handleMarkedEngineDirs();
```

Comment:
ここも元に戻せそう！（別にこのままでも問題ないですが）

```suggestion
      return this.vvppManager.handleMarkedEngineDirs();
```